homeassistant/components/husqvarna_automower/sensor.py

```diff
@@ -26,22 +26,22 @@
 
 
 @dataclass(frozen=True, kw_only=True)
-class AutomowerSensorEntityDescription(SensorEntityDescription):
+class AutomowerBinarySensorEntityDescription(SensorEntityDescription):
     """Describes Automower sensor entity."""
 
     exists_fn: Callable[[MowerAttributes], bool] = lambda _: True
     value_fn: Callable[[MowerAttributes], str]
 
 
-SENSOR_TYPES: tuple[AutomowerSensorEntityDescription, ...] = (
-    AutomowerSensorEntityDescription(
+SENSOR_TYPES: tuple[AutomowerBinarySensorEntityDescription, ...] = (
```

Comment:
I think you used the refactor button and forgot to check what else it changed

Reply:
I searched and replaced. Thanks. I changed it.